doc/classes/Node.xml

```diff
@@ -967,6 +967,13 @@
 				Similar to [method call_thread_safe], but for setting properties.
 			</description>
 		</method>
+		<method name="set_translation_domain_inherited">
+			<return type="void" />
+			<description>
+				Makes this node inherit translation domain from its parent node. The main translation domain will be used when this node has no parent.
+				This is the default behavior for all nodes. Calling [method Object.set_translation_domain] disables this behavior.
```

Comment:
A bit awkward that this method does not require an argument (such as `true` or `false`) like many others. Dare I say, prefixing the method with `set_` contributes to the confusion. But I digress.

I was going to suggest more but I am currently unsure, just generally unsure.

```suggestion
				Makes this node inherit the translation domain from its parent node. If this node has no parent, the main translation domain will be used.
				This is the default behavior for all nodes. Calling [method Object.set_translation_domain] disables this behavior.
```

Reply:
`set_translation_domain_inherited()` and `set_translation_domain(name)` is a pair of related operations.

Turning inheritance on is just... on. But turning it off requires specifying the fixed translation domain name at the same time.

The name of `set_translation_domain(name)` does not mention it disables inheritance because it's a virtual method of `Object`. Inheritance is a concept added in `Node`.